test/extensions/quic_listeners/quiche/active_quic_listener_test.cc

```diff
@@ -42,8 +43,11 @@
 #include "gmock/gmock.h"
 #include "extensions/quic_listeners/quiche/active_quic_listener_config.h"
 #include "extensions/quic_listeners/quiche/platform/envoy_quic_clock.h"
+#include "extensions/quic_listeners/quiche/envoy_quic_packet_writer.h"
 #include "extensions/quic_listeners/quiche/envoy_quic_utils.h"
+#ifndef WIN32
 #include "extensions/quic_listeners/quiche/udp_gso_batch_writer.h"
+#endif
```

Comment:
I thought @ggreenway had fixed this so we don't need ifdef for this anymore, at least in the main code. Is there any way to avoid the ifdef in this test?

Reply:
IIUC this specific test does validate udp_gso_batch_writer by name, not the generic wrapper which is windows agnostic, but I could be wrong; @ggreenway ?

Reply:
Optimally we would not have this type of ifdef/dependency in the test. IMO I would just remove it entirely and do whatever you did on Windows. In the interest of moving this forward I would be fine with a TODO and a follow up?

Reply:
The change that removed the need for this ifdef isn't merged yet. It's #13086

Reply:
(Modulo an error I've introduced and need to fix)

/wait

Reply:
Fix should be in... waiting on CI confirmation

Reply:
This is resolved. Note the introduction of new build failures. Because these are batch writer related, we are marking that test failing on windows, for the time being, as it's a regression in the most recent batch writer refactoring. Let's get this merged to avoid future breakage and we'll need to unwind the two broken tests.